### PR TITLE
Updated readme github buttons

### PR DESCRIPTION
**Pull request by swathiswaminathan**
Added a 'open in cloudshell' feature into the Readme

### CODE REVIEW

**Review by chrislovecnm — APPROVED** [review]
Thanks

File: README.md [hunk 1]
```diff
@@ -80,6 +80,21 @@ available in the Stackdriver Trace Console.
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```
Comment (chrislovecnm) [inline README.md]: Please update

**Review by chrislovecnm — CHANGES_REQUESTED** [review]

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Review by emailbob — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (emailbob) [reply README.md]: This has been updated

**Review by chrislovecnm — APPROVED** [review]

**Pull request merged**